assignments/function-conversion.js

```diff
@@ -5,26 +5,55 @@
 // };
 // myFunction();
 
+ let myFunction = () => {
+    console.log("Function was invoked!");
+};
+myFunction();
+
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
 
+let anotherFunction = (param) => {
+    return param;
+};
+anotherFunction("Example");
+
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
 
+let add = (param1, param2) => {
+    return param1 + param2;
+};
+add(1,2);
+
+
+
 // let subtract = function (param1, param2) {
 //   return param1 - param2;
 // };
 // subtract(1,2);
 
+let subtract = (param1, param2) => {
+    return param1 - param2;
+};
+subtract(1,2);
+
+
 
 // Stretch
 
 // exampleArray = [1,2,3,4];
 // const triple = exampleArray.map(function (num) {
 //   return num * 3;
 // });
-// console.log(triple);
\ No newline at end of file
+// console.log(triple);
+
+exampleArray = [1,2,3,4];
+const triple = exampleArray.map (function (num) => {
```

Comment:
Don't forget to remove the `function` keyword _or_ the arrow here!

Reply:
Oh yea. lol